discussions/discussion06/discussion6.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
<p>Need to fix the latex rendering.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tools4ds/ds701_fa2024_assignments/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Made it work</p>